;;; EXE: test/run-d8.py
;;; FLAGS: --spec
;;; STDIN_FILE: third_party/spec/ml-proto/test/memory.wast
(;; STDOUT ;;;
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:13:21: max size (0) must be greater than or equal to initial size (1)
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:17:34: segment ends past the end of the initial memory size (0)
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:21:53: address (500) greater than initial memory size (100)
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:25:55: address (0) less than end of previous segment (3)
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:29:54: address (0) less than end of previous segment (5)
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:33:69: address (1) less than end of previous segment (3)
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:44:29: alignment must be power-of-two
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:48:29: alignment must be power-of-two
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:52:29: alignment must be power-of-two
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:56:29: alignment must be power-of-two
assert_invalid error:
  third_party/spec/ml-proto/test/memory.wast:60:29: alignment must be power-of-two
24/24 tests passed.
;;; STDOUT ;;)
